roles/common/defaults/main.yml

```
@@ -1,7 +1,6 @@
 ---
 # defaults file for common
-os_path: $PATH
-log_directory: /var/log/ripu
-log_file: "{{ log_directory }}/ripu.log"
-
+leapp_os_path: $PATH
```

```suggestion
leapp_os_path: "{{ os_path | default('$PATH') }}"
```
?  Unless I'm missing something?

---

Common role is used from within other roles only so it doesn't require to be pre-populated.